services/frontend-v3/src/components/profileForms/employmentDataForm/EmploymentDataFormView.jsx

```diff
@@ -217,21 +227,28 @@ const EmploymentDataFormView = ({
     return undefined;
   };
 
-  /* show message */
-  const openNotificationWithIcon = (type) => {
+  /*
+   * Open Notification
+   *
+   * open notification to show status to user
+   */
+  const openNotificationWithIcon = ({ type, description }) => {
```

Comment:
I think these types of comments hinder code readability (and maintability - since people need to update them). As long as the function/variable name explains well what is does/contains, we should probably not put comments for them. What do you think about that?

Here (since it doesn't look like we're gonna be moving to typescript - at least for the frontend), putting JSdocs for the params/return instead would be more useful. (Little side note, for comments to be seen when using a function/variable, they need to start with /**, to create a JSdoc comment).

Also, I'm curious as to why you passing an object to this function instead of two distinct param.

----
Ex (if it would be two params):
```suggestion
  /**
    * @param {"success" | "error" | "warning"} type Notification type
    * @param {string} description Text to be displayed in the notification
    */
  const openNotificationWithIcon = (type, description) => {
```

which would render like this in vscode when using/hovering the function
![image](https://user-images.githubusercontent.com/22248828/90150640-e14a7a00-dd53-11ea-9761-619fd569a271.png)
![image](https://user-images.githubusercontent.com/22248828/90150861-15259f80-dd54-11ea-8fbf-a290f3751e33.png)


instead of 
![image](https://user-images.githubusercontent.com/22248828/90150524-c11abb00-dd53-11ea-8fb1-8ea4c937221b.png)

Reply:
on the topic of the comment. I was just trying to be consistent with the rest of the code. If we choose to change it  then we should do it for all functions in those files. I agree with  JSdocs for the params/return.

with regards to the object as the input param of the function, my reasoning was similar to what this guy is saying in this article. But I can change it. I dont mind.
https://medium.com/@david.gilbertson/named-function-arguments-61b9f88bd5e5